### PR TITLE
localstorage

### PR DESCRIPTION
**Pull request by rileylnapier**
## Description

[Pull Request Description Here]

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: packages/react-toast/src/components/Toast/index.tsx [hunk 1]
```diff
@@ -23,39 +24,38 @@ export const Toast: React.FunctionComponent<{
...
 
   const handleToast = useCallback(
     (message: ICourierToastMessage | string) => {
       message =
         typeof message === "string"
           ? ({
-            body: message,
-            icon: undefined,
-          } as ICourierToastMessage)
+              body: message,
```
Comment (suhasdeshpande) [inline packages/react-toast/src/components/Toast/index.tsx:34]: extra spacing

File: packages/react-inbox/src/components/Messages/index.tsx [hunk 2]
```diff
@@ -67,16 +71,21 @@ const Messages: React.FunctionComponent<InboxProps> = ({
...
             <Message key={message.messageId} {...message} />
-          ),
+          )
         )}
         {isLoading && <Loading />}
         {!isLoading && messages.length === 0 && (
           <Empty>You have no notifications at this time</Empty>
         )}
-        {!isLoading && messages.length && !startCursor && (<PaginationEnd title="End Of The Road" />) }
+        {!isLoading && messages.length > 5 && !startCursor && (
```
Comment (rileylnapier) [inline packages/react-inbox/src/components/Messages/index.tsx:80]: im not too sure about this logic here, but i think we only want to show the end of the road if we have > 5 messages

File: packages/react-toast/src/components/Toast/index.tsx (hunk 1 above)
Comment (rileylnapier) [reply packages/react-toast/src/components/Toast/index.tsx:34]: we need to get the linter right on this stuff.

File: packages/react-inbox/src/components/Inbox/index.tsx [hunk 3]
```diff
@@ -82,8 +84,46 @@ const Inbox: React.FunctionComponent<InboxProps> = (props) => {
...
+          console.log("error", ex);
+          // do nothing
+        }
+      }
+    }
+
+    initInbox({
+      config: props,
+    });
+  }, [props, clientKey, userId]);
```
Comment (RoyalSix) [inline packages/react-inbox/src/components/Inbox/index.tsx:109]: Should this be getting called everytime the props object changes?

Comment (rileylnapier) [reply packages/react-inbox/src/components/Inbox/index.tsx:109]: yeah, because we need to update the context

**Review by RoyalSix — APPROVED** [review]

**Pull request merged**